notebooks/milestone1.ipynb

```diff
@@ -615,44 +899,191 @@
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df.model"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 3.1 Load the entire data "
+    "### 3.1 Load only the interested columns"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 42,
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "MPI-ESM1-2-HR       5154240\n",
+      "NorESM2-MM          3541230\n",
+      "CMCC-CM2-SR5        3541230\n",
+      "CMCC-CM2-HR4        3541230\n",
+      "CMCC-ESM2           3541230\n",
+      "TaiESM1             3541230\n",
+      "SAM0-UNICON         3541153\n",
+      "GFDL-ESM4           3219300\n",
+      "GFDL-CM4            3219300\n",
+      "FGOALS-f3-L         3219300\n",
+      "MRI-ESM2-0          3037320\n",
+      "EC-Earth3-Veg-LR    3037320\n",
+      "BCC-CSM2-MR         3035340\n",
+      "MIROC6              2070900\n",
+      "ACCESS-CM2          1932840\n",
+      "ACCESS-ESM1-5       1610700\n",
+      "INM-CM4-8           1609650\n",
+      "INM-CM5-0           1609650\n",
+      "KIOST-ESM           1287720\n",
+      "FGOALS-g3           1287720\n",
+      "MPI-ESM1-2-LR        966420\n",
+      "NESM3                966420\n",
+      "AWI-ESM-1-1-LR       966420\n",
+      "MPI-ESM-1-2-HAM      966420\n",
+      "NorESM2-LM           919800\n",
+      "CanESM5              551880\n",
+      "BCC-ESM1             551880\n",
+      "Name: model, dtype: int64\n",
+      "peak memory: 1062.32 MiB, increment: 958.63 MiB\n",
+      "Wall time: 48 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "%%memit\n",
+    "#48s\n",
+    "\n",
+    "data_model = pd.read_csv(out_combined_filepath, usecols=['model'])\n",
+    "print(data_model.model.value_counts())\n"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 3.2 Load only the interested columns"
+    "### 3.2 Change `dtype` of the data"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": 36,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "### 3.3 Change `dtype` of the data"
+    "f64_mem = df.iloc[:, 1:6].memory_usage().sum()/1e6\n",
+    "f32_mem = df.iloc[:, 1:6].astype('float32', errors='ignore').memory_usage().sum() / 1e6"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": 37,
    "metadata": {},
-   "source": []
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Memory usage with float64:  2498.71 MB\n",
+      "Memory usage withdf float64:  1249.36 MB\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"Memory usage with float64: {f64_mem: .2f} MB\")\n",
+    "print(f\"Memory usage withdf float64: {f32_mem: .2f} MB\")"
```

Comment:
There might be a typo here: the second print argument should be:`print(f\"Memory usage with float32: {f32_mem: .2f} MB\")`